servers/src/test/java/tachyon/master/file/meta/InodeFileTest.java

```diff
@@ -79,6 +80,28 @@ public void getBlockSizeBytesTest() {
   }
 
   @Test
+  public void getBlockIdByIndexTest() throws Exception {
+	  InodeFile inodeFile = createInodeFile(1);
+	  // create 3 block ids
+	  inodeFile.getNewBlockId();
```

Comment:
Consider using a for loop for this:

```
final int NUM_BLOCKS = 3;
for (int i = 0; i < NUM_BLOCKS; i ++) {
  long blockId = inodeFile.getNewBlockId();
  Assert.assertEquals(blockId, inodeFile.getBlockIdByIndex(i));
}
```
and use NUM_BLOCKS below instead of hard coding 3 into the string constant